src/main/java/com/mindera/pizza/services/order/RestaurantOrderService.java

```diff
@@ -49,4 +52,15 @@ public RestaurantOrder createOrder(CreateRestaurantOrderDTO restaurantOrderDTO)
 
         return restaurantOrderRepo.save(restaurantOrder);
     }
+
+    public List<RestaurantOrder> findOrders(Map<String, String> filters) {
+        Specification<RestaurantOrder> spec = (root, query, criteriaBuilder) -> criteriaBuilder.conjunction();
+
+        for (Map.Entry<String, String> entry : filters.entrySet()) {
+            Specification<RestaurantOrder> currentSpec = RestaurantOrderSpecifications.getSpecificationFromFilterName(entry.getKey(), entry.getValue());
+            if (currentSpec != null) spec = spec.and(currentSpec);
```

Comment:
convém ser consistente, se normalmente usas a estrutura de colocar '{' e '}' nas condições de if, for, etc é bom manter os {}